components/Activity/menu.ts

```diff
@@ -4,4 +4,11 @@ import { MenuItem } from '../User/SessionBox';
 export const organizerMenu = ({ t }: typeof i18n, activityId: number): MenuItem[] => [
   { href: `/activity/${activityId}/editor`, title: t('edit_activity') },
   { href: `/activity/${activityId}/forum`, title: t('forum_list') },
+  { href: `/activity/${activityId}/cooperation`, title: t('cooperation_management') },
+];
+
+export const userMenu = ({ t }: typeof i18n): MenuItem[] => [
+  { href: '/organization', title: t('organization_list') },
+  { href: '/activity', title: t('activity_list') },
+  // { href: '/session', title: t('session_list') }, // TODO: Add when session management is implemented
```

Comment:
```suggestion
  { href: '/user/organization', title: t('organization_list') },
  { href: '/user/activity', title: t('activity_list') },
  { href: '/user/session', title: t('session_list') },
```
路径不涉及直接的运行时引用，可以先写上。